packages/linter/src/builders/lint/schema.json

```diff
@@ -28,23 +28,24 @@
     },
     "format": {
       "type": "string",
-      "description": "Output format (prose, json, stylish, verbose, pmd, msbuild, checkstyle, vso, fileslist).",
-      "default": "prose",
+      "description": "ESLint Output formatter (https://eslint.org/docs/user-guide/formatters).",
```

Comment:
I guess we had a mismatch before since tslint and eslint have different formatters. :)

Reply:
I believe it worked though because we were explicitly deleting `options.format` before sending it off to `angular-eslint/builder` - which had the proper formatters
and we are not supporting `tslint` with this builder